README.md

```diff
@@ -298,6 +298,12 @@ asyncio.run(main())
 
 </details>
 
+## Known issues
+
+### `time.sleep()` leads to outdated state
+
+We internally rely on asynchronous operations. When using `time.sleep(5)` they can't be processed internally and by that e.g. the internal storage for pages can't get updated. To fix this behaviour you should use `page.wait_for_timeout(5000)` instead if you are using the synchronous api. For the asynchronous api you can use `await asyncio.sleep(5)`.
```

Comment:
I would start with the solution:
===
Always use page.wait_for_timeout(5000) instead of thread.sleep(5).

It is better to not wait for timeout at all, but sometimes it is useful for debugging. In these cases, use our wait method instead of the system one.
===